app/components/Notification/index.jsx

```diff
@@ -54,7 +66,7 @@ let NotifiItem = React.createClass({
                         <div className="name" style={nameStyle}><Name member={this.props.sender}/></div>
                         <div className="time" style={timeStyle} flex={1}><SmartTimeDisplay start={this.props.time} relative/></div>
                     </Layout>
-                    <div style={messageStyle}>{this.props.message}</div>
+                    <span style={achorStyle} onClick={this._transitionToPost}>{this.props.message}</span>
```

Comment:
FYI, you can use https://github.com/rhinobird-io/rhinobird-web/blob/master/app/components/Common/RouterLink.jsx